clients/gtest/axpy_ex_gtest.cpp

```diff
@@ -157,7 +161,7 @@ TEST_P(axpy_ex_gtest, axpy_batched_ex)
         }
         else
         {
-            EXPECT_EQ(HIPBLAS_STATUS_NOT_SUPPORTED, status); // for CUDA
+            EXPECT_EQ(HIPBLAS_STATUS_SUCCESS, status); // fail
```

Comment:
Would this not pass , what does // fail mean?
EXPECT_EQ( defined(__HIP_PLATFORM_NVCC__) ? HIPBLAS_STATUS_NOT_SUPPORTED :  HIPBLAS_STATUS_SUCCESS, status );

Reply:
If I removed the `ifndef`, the code you suggested wouldn't work by itself. In testing_xx.hpp, when it reaches the first `CHECK_HIPBLAS_ERROR()` from a `_batched` call, there will be an error as on NVCC we will not be returning success. We could potentially change the macro to pass on NVCC if we get a `HIPBLAS_STATUS_SUCCESS` OR `HIPBLAS_STATUS_NOT_SUPPORTED`, but that could potentially hide issues as well.

The `// fail` comment means that if we get to that point in the code, the test will be failing. The `if` statement a couple lines prior avoids this code if we returned a success, this could be changed a bit/made more clear, but I won't do that here.